src/pheromones/CMakeLists.txt

```diff
@@ -104,10 +111,9 @@ add_library(${LIB_ALIAS} ALIAS ${LIB_NAME})
 target_include_directories(${LIB_NAME}
     PUBLIC
        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-       $<INSTALL_INTERFACE:include>
-
-        #Include the generated files from .in
-        ${CMAKE_CURRENT_BINARY_DIR}/include
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include>
```

Comment:
I don't know how pick you are with cmake file indent, but one space too much here